lib/core/resolvers/GitResolver.js

```diff
@@ -26,6 +26,9 @@ function GitResolver(decEndpoint, config, logger) {
     // anyway
     mkdirp.sync(config.storage.empty);
     process.env.GIT_TEMPLATE_DIR = config.storage.empty;
+    if (!config.strictSsl) {
+        process.env.GIT_SSL_NO_VERIFY = 'true';
+    }
```

Comment:
Sorry, I dont get what should be in "else" part.
How is it different from  ``process.env.GIT_TEMPLATE_DIR`` line above?